app/src/webtrigger/handle-jenkins-request.ts

```diff
@@ -119,8 +133,15 @@ async function getGatingStatus(cloudId: string, request: GatingStatusRequest): P
 
 function logJiraResponse(jiraResponse: JiraResponse) {
 	if (jiraResponse.status >= 400) {
-		// eslint-disable-next-line no-console,max-len
-		console.error(`Received response with status ${jiraResponse.status} from Jira: ${JSON.stringify(jiraResponse.body)}`);
+		log(
+			'logJiraResponse',
+			LogLevel.ERROR,
+			{
+				eventType: 'logJiraResponseEvent',
+				errorMsg: `Received error response from Jira - status: ${jiraResponse.status}`,
+				error: JSON.stringify(jiraResponse.body)
```

Comment:
it may already be logged elsewhere so ignore if thats the case, but i do very much like status being a attribute on these logs, makes life easier in Splunk